### ISSUE TITLE
[Bug]: `pest()->printer()` does not exists

### ISSUE THREAD

**Issue by schonhoff** [pestphp/pest#1229]
### What Happened

Hello,

I'm currently upgrading to Pest v3. After I read the upgrade guide and the "Pest v3 Now Available" page I tried to switch from using
`uses()->compact();` to `pest()->printer()->compact();`
This was mentioned here:
[https://pestphp.com/docs/pest3-now-available#content-new-configuration-api](url)

After switching my PHPStorm was showing a warning and running a test will throw this error:
```batch
 Error 

  Call to undefined method Pest\PendingCalls\UsesCall::printer()

  at vendor\pestphp\pest\src\Configuration.php:112
    108▕      * @param  array<array-key, mixed>  $arguments
    109▕      */
    110▕     public function __call(string $name, array $arguments): mixed
    111▕     {
  ➜ 112▕         return $this->uses()->$name(...$arguments); // @phpstan-ignore-line
    113▕     }
    114▕ }
    115▕

  1   tests\Pest.php:14
      Pest\Configuration::__call("printer", [])
  2   vendor\pestphp\pest\src\Bootstrappers\BootFiles.php:76
```

Thanks for the great work and I love the new arch presets.

### How to Reproduce

see "What Happened"

### Pest Version

3.0.2

### PHP Version

8.3.8

### Operation System

Windows

**Comment by edwinheij** [comment]
Just use the ```theme``` method instead of ```printer```

**Comment by edwinheij** [comment]
made a PR for the docs just now

**Comment by nunomaduro** [comment]
It's `pest()->printer()`: fixed on release: `v3.0.3`.
